rdkit-stubs/Chem/BuildFragmentCatalog.pyi

```diff
@@ -1,98 +1,204 @@
 from _typeshed import Incomplete
+from collections.abc import Sequence
+from io import IOBase, TextIOWrapper
+import sys
+from typing import Optional
+import numpy as np
+import numpy.typing as npt
 from rdkit import RDConfig as RDConfig
 from rdkit.Chem import FragmentCatalog as FragmentCatalog
 from rdkit.Dbase.DbConnection import DbConnect as DbConnect
 from rdkit.ML import InfoTheory as InfoTheory
 
-def message(msg, dest=...) -> None: ...
+def message(msg, dest=sys.stdout) -> None: ...
 def BuildCatalog(
     suppl,
-    maxPts: int = ...,
-    groupFileName: Incomplete | None = ...,
-    minPath: int = ...,
-    maxPath: int = ...,
-    reportFreq: int = ...,
-): ...
+    maxPts: int = -1,
+    groupFileName: Optional[str] = None,
```

Comment:
Would be ok to leave as `Incomplete | None` I think but it's ok either way.

Reply:
I checked the meaning of `Incomplete` on this typeshed. According to the issue, `Incomplete` is suggested with options of `Unknown` `Something` `Untyped` `Unannotated`. Now I have checked that the variable `groupFileName` should be `str` or `None` thus I think it is better to use `Optional[str]`. `str | None` is a second choice, and both of them are okay for me.

Reference: https://github.com/python/typeshed/issues/5613

Reply:
Oh, my mistake. I meant `str | None`, not `Incomplete | None`.